doc/en/how-to/tmp_path.rst

```diff
@@ -155,4 +155,14 @@ When distributing tests on the local machine using ``pytest-xdist``, care is tak
 automatically configure a basetemp directory for the sub processes such that all temporary
 data lands below a single per-test run basetemp directory.
 
+
+.. _`file permissions`:
+
+File permissions
+----------------
+
+Any file or directory created by the above fixtures are by default created with private permissions (file mask 700).
+
+You can override the file mask by setting the ``PYTEST_TMPDIR_FILE_MASK`` environment variable.
```

Comment:
Please also add a reference to this new env var in `reference.rst`, under the `Environment Variables` section. Then we can link to it here:

```suggestion
You can override the file mask by setting the :envvar:`PYTEST_TMPDIR_FILE_MASK` environment variable as an octal string, the default being `0o700`.
```

Reply:
Also it would help to mention here that this env var is useful mostly for the container sharing situation described in #10738,

Reply:
Would not make more sense to call this `PYTEST_TMPDIR_FILE_MODE` instead, given this provides directly the mode?